.changeset/mean-horses-kiss.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Would love to see an example of an integration using the hook in here

Reply:
Isn't it already the case with the snippet below?

Reply:
I mean in the actual integration hook, the snippet doesn't show where that function comes from. You can amend the snippet below too, no need for a new example necessarily